brave/brave-core
# Add pages and domains loaded P3A metrics

Resolves brave/brave-browser#25867

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

0. Start browser with fresh profile. Ensure metrics have a value of 0 on local state page.
1. Load some random pages, ensure to keep count of pages loaded and unique domains loaded.
2. Close & open browser, wait 30 seconds. Ensure recorded value for `Brave.Core.PagesLoaded` is correct given the amount of pages loaded.
3. Close browser, move system time forward by one day, open browser, wait 30 seconds. Ensure `Brave.Core.DomainsLoaded` is correct given the amount of unique domains loaded.
4. Close browser, move system time forward by seven days, open browser, wait 30 seconds. Ensure both metrics have a value of 0.
